src/scripts/movies.js

```diff
@@ -33,9 +48,14 @@ define(["layoutManager", "userSettings", "events", "libraryBrowser", "alphaPicke
                     filterButton: !1
                 }),
                 elems = tabContent.querySelectorAll(".paging");
-            for (i = 0, length = elems.length; i < length; i++) elems[i].innerHTML = pagingHtml;
-            for (elems = tabContent.querySelectorAll(".btnNextPage"), i = 0, length = elems.length; i < length; i++) elems[i].addEventListener("click", onNextPageClick);
-            for (elems = tabContent.querySelectorAll(".btnPreviousPage"), i = 0, length = elems.length; i < length; i++) elems[i].addEventListener("click", onPreviousPageClick)
+            for (i = 0, length = elems.length; i < length; i++)
```

Comment:
```suggestion
            for (elems = tabContent.querySelectorAll(".paging"), i = 0, length = elems.length; i < length; i++)
```
to be consistent with other `for`-s in this function